processor/src/main/resources/org.mapstruct.ap.model.NestedPropertyMappingMethod.ftl

```diff
@@ -24,7 +24,7 @@
         return null;
     }
     <#list propertyEntries as entry>
-    <@includeModel object=entry.type/> ${entry.name} = <#if entry_index == 0>${sourceParameter.name}.${entry.accessor}<#else>${propertyEntries[entry_index-1].name}.${entry.accessor}</#if>;
+    <@includeModel object=entry.type/> ${entry.name} = <#if entry_index == 0>${sourceParameter.name}.${entry.accessorName}()<#else>${propertyEntries[entry_index-1].name}.${entry.accessorName}()</#if>;
```

Comment:
Interesting, I'm wondering why this was not always a problem? I guess I still must be missing something.